### ISSUE TITLE
Fred in articles needs to be fixed

### ISSUE THREAD

**Issue by svenoe** [RotherOSS/fred#3]
At least in the customer interface fred has to be fixed in the articles - maybe in all places where we call OTOBO out of an iframe. I suppose this is because of the switch to psgi and how headers are handled. I didn't look into this at all yet, really...

**Comment by bschmalhofer** [comment]
Hi @svenoe,

I started to look into this, but I'm not sure what the expected behaviour is. 

For testing I installed Fred in a fresh OTOBO installation. Then I added an internal note to the welcome ticket. The effect was that I saw the Fred output, a bit differently presented, in the iframe for the note. The intended content of the not was not visible. 

What should have happened? Fred not being shown in the iframe or Fred nicely formatted in the iframe?

![Screenshot 2022-03-09 at 15-58-27 2015071510123456 - Inhalt - Ticket - OTOBO 10](https://user-images.githubusercontent.com/46019/157468636-5b172c75-04c4-4906-bd34-860d89a0c2fa.png)
